pymeasure/instruments/temptronic/temptronic_base.py

```diff
@@ -0,0 +1,841 @@
+        NOTE: The above bits are latched, and are automatically cleared
+        when the register is read.
+        """,
+        validator=truncated_range,
+        values=[0,255]
+        )
+
+    standard_event_status_mask = Instrument.control(
+        "ESE?", "ESE %g",
+        """Read the standard event status register.
+
+        bit 7 – power on – not used
+        bit 6 – user request -- not used
+        bit 5 – command error (cme)
+        bit 4 – execution error (exe)
+        bit 3 – device dependent error (dde)
+        bit 2 – query error (qye)
+        bit 1 – request control -- not used
+        bit 0 – operation complete -- not implemented
+        NOTE: The above bits are latched, and are automatically cleared
+        when the register is read.
+        """,
+        validator=truncated_range,
+        values=[0,255]
+        )
+
+    service_request_mask = Instrument.control(
+        "SRE?", "SRE %g",
+        """Set the service request enable (mask) register.
+
+        SRE nnn -- where nnn is 0 – 255
+        bit 7 – ready
+        bit 6 – request for service (RQS)
+        bit 5 – standard event status (ESB) summary bit
+        bit 4 – message available (MAV) (always 0 for RS-232)
+        bit 3 – temperature event (TESR) summary bit
+        bit 2 – device specific error (EROR) summary bit
+        bit 1 – not used (always 0)
+        bit 0 – not used (always 0)
+        NOTE: See <serial poll> for the meaning of each bit in the mask.
+        Read the service request enable (mask) register.
+        """,
+        validator=truncated_range,
+        values=[0,255]
+        )
+
+    local_lockout = Instrument.setting(
+        "%LL",
+        """Local lockout – the System touch screen controls are disabled, and
+        no “Return to local” button appears on the panel.
+        """
+        )
+
+
+    system_current = Instrument.measurement(
+        "AMPS?",
+        """Read the system’s current draw (amperage).
+        """,
+        )
+
+
+    auxiliary_condition_code = Instrument.measurement(
+        "AUXC?",
+        """Read the auxiliary condition register.
+
+        bit 10 – reserved
+        bit 9 – ramp mode
+        bit 8 – Manual Mode= 1, Program= 0
+        bit 7 – reserved
+        bit 6 – ready for operation = 1, startup sequence = 0
+        bit 5 – flow on = 1, flow off = 0
+        bit 4 – DUT mode = 1, air-control mode = 0
+        bit 3 – heat only mode = 1, compressor on =0
+        bit 2 – head up = 1, head down = 0
+        bit 1 – reserved
+        bit 0 – reserved
+        """,
+        validator=truncated_range,
+        values=[0, 1023]
+        )
+ 
+    copy_active_setup_file = Instrument.setting(
+        "CFIL %g",
+        """Local lockout – the System touch screen controls are disabled, and
+        no “Return to local” button appears on the panel.
+        """,
+        validator=strict_range,
+        values=[1, 12]
+        )
+
+    clear_errors = Instrument.setting(
```

Comment:
I overlooked this previously, now with [this commit](https://github.com/BoschSensortec/pymeasure/commit/9f92b364308bffa6e508d1e686aad81ea79f3dd6) it will also be fixed.